plugins/repository-s3/build.gradle

```diff
@@ -52,7 +52,7 @@ dependencies {
   // and whitelist this hack in JarHell
   compile 'javax.xml.bind:jaxb-api:2.2.2'
 
-  testCompile project(':test:fixtures:minio-fixture')
+  testCompile project(':test:fixtures:s3-fixture')
```

Comment:
would be nice to be able to avoid this, but as we discussed in one of the other PRs we can do that separately.

Reply:
What's the concern about having this additional test dependency?

Reply:
@mark-vieira We discussed this point in https://github.com/elastic/elasticsearch/pull/48636#discussion_r340434656.

Reply:
I think this is fine.

Reply:
Thanks @mark-vieira for your feedback :+1: